release-notes/opensearch-dashboards.release-notes-1.3.6.md

```diff
@@ -0,0 +1,25 @@
+## Version 1.3.6 Release Notes
+
+### 📈 Features/Enhancements
+
+* Custom healthcheck with filters ([#2232](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2232), [#2277](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2277))
```

Comment:
I think historically we might have given something insight how to configure it here. Did we want to try this? By saying set with:
```
opensearch.optimizedHealthcheck.filters: {
   attribute_key: "attribute_value",
}
```

Reply:
It feels a little verbose for release notes, so I linked to the example config instead.